cmd/resize.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+    "errors"
+
+    "github.com/getcarina/carina/console"
+    "github.com/spf13/cobra"
+)
+
+func newResizeCommand() *cobra.Command {
+    var options struct {
+        name     string
+        nodes    int
+        wait     bool
+    }
+
+    var cmd = &cobra.Command{
+        Use:               "resize <cluster-name>",
+        Short:             "Resize a cluster",
+        Long:              "Resize a cluster",
+        Hidden:            true,
+        PersistentPreRunE: authenticatedPreRunE,
+        PreRunE: func(cmd *cobra.Command, args []string) error {
+            if options.nodes < 1 {
+                return errors.New("--nodes must be >= 1")
+            }
+
+            return bindClusterNameArg(args, &options.name)
+        },
+        RunE: func(cmd *cobra.Command, args []string) error {
+            cluster, err := cxt.Client.ResizeCluster(cxt.Account, options.name, options.nodes, options.wait)
+            if err != nil {
+                return err
+            }
+
+            console.WriteCluster(cluster)
+
+            return nil
+        },
+    }
+
+    cmd.ValidArgs = []string{"cluster-name"}
+    cmd.Flags().IntVar(&options.nodes, "nodes", 1, "Number of nodes to add to the cluster")
+    cmd.Flags().BoolVar(&options.wait, "wait", false, "Wait for the cluster to become active")
```

Comment:
Just a thought that the resizing state is funny, because while the cluster isn't official in the `ACTIVE` state, it is quite active/usable from the user's point of view. Not sure I want to try to explain details like that though in the cli help text. Maybe something worth mentioning in our online documentation?